### DISCUSSION TITLE
Questions about the projects (and themes)

### DISCUSSION THREAD

**Discussion by ghost** [olive-editor/olive#1937] [General]
# Question
Hello! If you're seeing this and you're a part of the Olive developer team, I have some questions for you!

### Questions

1. Do you plan on ever adding themes? Light mode, custom themes, etc.
2. How large is your developer team, not including contributors?
3. Do you have a road-map for Olive?
4. Do you plan on gaining more developers?

Thank you!

Btw you guys are AWESOME!!

**Comment by Simran-B** [comment]
1. There are two themes already, a dark and a light theme (under **Appearance** in the preferences). It's also possible to add custom themes, but it requires a few code changes and thus to compile Olive from source to add your own. There is currently no intention to make theming easier. Personally, I think that themes should serve an actual purpose, such as reducing eye strain (dark theme), or to enable people with vision deficiencies to work with the application at all (e.g. a high contrast theme), but not just out of a desire to customize for taste.
2. Basically one, the creator himself, working on the project part-time, with the desire to make it a full-time job if funding allows. There is one more frequent code contributor who belongs to the core team, another guy and me are also in the team but don't really code. Everything else I would consider occasional, external contributions. If you want to take a look at the statistics: https://github.com/olive-editor/olive/graphs/contributors
3. No clear roadmap, but the current focus is on the ability to edit, stability, and performance. Also see https://www.patreon.com/posts/olive-april-2022-66179918, especially the comments.
4. Volunteers are always welcome, and it would nice to also hire developers but the project doesn't have the necessary funding for that at the moment.

**Reply by ghost** [reply]
I would love to volunteer in the future and work on the UI. (if that's okay, of course)
I specialize in the C languages and would LOVE to help with UI if you want. What you are working on is amazing, and there truly isn't anything like it. I have experience with a crap ton of languages but like stated earlier, specialize in the C languages and Python. Look forward to contributing in the future! You guys are amazing!

**Reply by Simran-B** [reply]
Thank you for the kind words! The UI isn't a priority at the moment but undoubtedly needs more love. You are welcome to join the [Discord server](https://discordapp.com/invite/4Ae9KZn) if you want to discuss ideas.

The codebase is in C++17 and makes heavy use of Qt. General programming knowledge is helpful, nonetheless. We have vague plans to adopt [Conan](https://conan.io/) as a package manager to make it easier to build Olive on all platforms, and it's written in Python. That would also be an opportunity for you if you are interested.

**Reply by ghost** [reply]
This is perfect, as I have experience with all of the above! I look forward to helping you guys out!

**Comment by pgilfernandez** [comment]
As reference, here https://github.com/olive-editor/olive/pull/1882 you can find my attempt to propose new styles
